src/errorHandler.ts

```diff
@@ -14,6 +14,7 @@ import { onWorkspaceRootChange, allEnabledLanguages } from './utils';
 let statusBarItem: StatusBarItem;
 let outputChannel: OutputChannel;
 let outputChannelOpen: Boolean = false;
+let prettierInformation: String;
```

Comment:
This should be `string` (lower case) and `boolean` above.
You can avoid toString() below.